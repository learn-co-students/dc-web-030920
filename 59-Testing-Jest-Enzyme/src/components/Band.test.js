import React from 'react'
import { shallow } from 'enzyme'
import Band from './Band'

const mockProps = {
  name: 'Aerosmith',
  id: 1,
  deleteBand: jest.fn()
}

describe('Band', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Band {...mockProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})