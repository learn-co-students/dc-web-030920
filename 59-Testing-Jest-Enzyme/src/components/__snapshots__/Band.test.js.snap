// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Band renders correctly 1`] = `
<div>
  <li>
    Aerosmith
  </li>
  <button
    onClick={[Function]}
  >
    Delete
  </button>
</div>
`;
